pkg/lightning/backend/local/localhelper_test.go

```diff
@@ -325,3 +325,40 @@ func TestStoreWriteLimiter(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestTuneStoreWriteLimiter(t *testing.T) {
+	limiter := newStoreWriteLimiter(100)
+	testLimiter := func(ctx context.Context, maxT int) {
+		var wg sync.WaitGroup
+		for i := 0; i < 10; i++ {
+			wg.Add(1)
+			go func(storeID uint64) {
+				defer wg.Done()
+				start := time.Now()
+				var gotTokens int
+				for {
+					n := rand.Intn(50)
+					if limiter.WaitN(ctx, storeID, n) != nil {
+						break
+					}
+					gotTokens += n
+				}
+				elapsed := time.Since(start)
+				maxTokens := 120 + int(float64(elapsed.Seconds())*float64(maxT))
+				// In theory, gotTokens should be less than or equal to maxT.
+				// But we allow a little of error to avoid the test being flaky.
```

Comment:
Did you find this test is flaky? I think the sleep-related functions guarantee that *at least* some duration it will block. So the `gotTokens` should always be less or equal than maxT * elapsed.

Reply:
Oh I see it's because out limiter "Allow burst of at most 20% of the writeLimit." We can change hardcoded 120 to 1.2*maxT then.